resources/lang/fr/community.php

```diff
@@ -22,50 +22,50 @@
     'support' => [
         'header' => [
             // size in font-size
-            'big_description' => "Vous aimez osu!?<br/>
-                                Supportez le développement d'osu! :D",
+            'big_description' => 'Vous aimez osu! ?<br/>
+                                Supportez son développement :D',
             'small_description' => '',
             'support_button' => 'Je veux supporter osu!',
         ],
 
-        'dev_quote' => "osu! est un free-to-play complet, mais le maintenir n'est pas gratuit. Entre le prix des serveurs et de la grande bande passante internationale, le temps passé à maintenir le système et la communauté, parvenir des prix pour les compétitions, répondre aux questions du support et généralement garder les gens heureux, osu! consomme une certaine somme d'argent! Oh, et n'oubliez pas le fait que nous le faisons sans pub ou partenariat avec des toolbars ou autres!
-                    <br/><br/>osu! est à largement développé par moi-même, pour cela vous me connaisez sûrement avec le surnom \"peppy\".
-                    J'ai du quitté mon travail à temps plein pour garder le rythme avec osu!,
+        'dev_quote' => "osu! est un free-to-play complet, mais le maintenir n'est pas gratuit. Entre le prix des serveurs et de la grande bande passante internationale, le temps passé à maintenir le système et la communauté, obtenir des prix pour les compétitions, répondre aux questions du support et généralement garder les gens heureux, osu! consomme une certaine somme d'argent! Oh, et n'oubliez pas le fait que nous le faisons sans pub ou partenariat avec des toolbars ou autres!
```

Comment:
est-ce que "free-to-play" est traduit en général ? J'ai l'impression qu'on devrait le traduire ici